chains/evm/GNUmakefile

```diff
@@ -12,6 +12,10 @@ snapshot-diff: ## Make a snapshot for a specific product.
 foundry: ## Install foundry.
 	foundryup --install v1.0.0
 
+.PHONY: forge-zksync
+forge-zksync: ## Install foundry.
```

Comment:
is it possible to change this to `foundry-zksync` to align naming with the `make foundry` call

Reply:
Perfect too 👍